### PR TITLE
Run bootstrap before linting on CI

### PR DESCRIPTION
**Pull request by Hypnosphi**
See https://storybooks.slack.com/archives/C4Q0S3MQR/p1503243475000043

I've added only those bootsrappers, lack of which produced issues for me. Maybe it's worth to add some more (or remove `bootstrap:docs`)

The issues I faced are:
1) In, #1676, the only reason we can't promote `import/extensions` to "error" level is that packages `highlight.js` and `fuse.js` are treated by this rule as local files unless they're actually installed
2) In #1696, I have to add dependencies to `eslint-plugin-flowtype`, `eslint-plugin-flowtype-errors`, and `flow-bin` to root package instead of `lib/components` so that they're installed with simple `yarn install` without bootstrapping

The main problem is that all those issues are discoverable only when it actually fails on CI (unless you run `yarn bootstrapp -- --reset` before each commit, after you made the changes)

### CODE REVIEW

**Comment by ndelangen** [comment]
I'm not opposed to this at all, but please state the reasons clearly in the PR, Slack messages get lost over time, and the message you link to does not give a comprehensive picture of why this is exactly needed, thanks 👍

**Comment by Hypnosphi** [comment]
I've added some clarifications

**Comment by Hypnosphi** [comment]
Using close colours for background and text leads to bad readabiity =(

screen shot 2017-08-31 at 03 01 53

**Comment by Hypnosphi** [comment]
Are those lines really needed?
screen shot 2017-08-31 at 03 04 03
screen shot 2017-08-31 at 03 05 34

File: addons/links/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ linkTo('Toggle') // Links to the first story in the 'Toggle' kind
 With that, you can link an event in a component to any story in the Storybook.
 
 -   First parameter is the the story kind name (what you named with `storiesOf`).
--   Second (optional) parameter is the story name (what you named with `.add`). If the second parameter is omitted, the link will point to the first story in the given kind.
+-     Second (optional) parameter is the story name (what you named with `.add`). If the second parameter is omitted, the link will point to the first story in the given kind.
```
Comment (ndelangen) [inline addons/links/README.md]: ?

Comment (Hypnosphi) [reply addons/links/README.md]: No idea =(

**Comment by Hypnosphi** [comment]
I'm currently working on enabling yarn workspaces, which should make this PR partially obsolete.

@ndelangen is there any solid reason why we bootstrap `docs` package separately from others?

**Comment by Hypnosphi** [comment]
In fact, even with #1810 there's still a sense in running bootstrap: it removes a bunch of `imports/no-unresolved` warnings which are present because of lack of `dist` folders. This can allow to raise this rule to error level as well

**Comment by ndelangen** [comment]
Well the reason for not bootstrapping `docs` along with core, is because often it's not needed.

I would like to be able to bootstrap sections individually.
So a developer can choose what to work on, and get some faster bootstrap.

BUt it's a nice to have, I do not oppose including docs into the default bootstrap, but it is unnecessary for 90% of the development that occurs currently. (And even if the code-change is in docs, often it can be worked on from github GUI or without running the docs app to begin with)

**Comment by Hypnosphi** [comment]
Let‘s measure how much does it add up to clean bootstrap time

**Comment by Hypnosphi** [comment]
Unfortunately, when I try to add `docs` to workspaces, the build command fails:
```
Error: Error: Cannot find module './addons/_template.jsx'.
    at render-page.js:31859:42
    at webpackContextResolve (render-page.js:31859:90)
    at webpackContext (render-page.js:31856:30)
    at render-page.js:25774:36
    at Array.forEach (<anonymous>:null:null)
    at module.exports (render-page.js:25744:27)
    at render-page.js:90:40
    at module.exports (render-page.js:31819:11)
    at Object.<anonymous> (render-page.js:89:2)
    at __webpack_require__ (render-page.js:30:30)

```

**Comment by Hypnosphi** [comment]
Superseded by #1934

**Pull request closed without merging**